src/assets/src/components/queue.tsx

```diff
@@ -306,6 +307,9 @@ function QueueAttendingJoined(props: QueueAttendingProps) {
                 <Card.Text>
                     <strong>Meeting Via</strong>: {meetingBackend.friendly_name} {!inProgress && changeMeetingType}
                 </Card.Text>
+                <Card.Text>
+                    <strong>Meet At</strong>: {props.queue.physLocation}
```

Comment:
```suggestion
                    <strong>Meet At</strong>: {props.queue.phys_location}
```
To work with other changes.